packages/components/src/components/step-group/step-group.stories.ts

```diff
@@ -44,240 +53,160 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return html`<div style="height:250px">${generateTemplate({ args })}</div>`;
   }
 };
 
 /**
- * Use the orientation attribute to set the axis of a step-group.
+ * Use the `size` attribute to set the size of the step-group.
  */
 
-export const Orientation = {
-  parameters: { controls: { exclude: 'orientation' } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'orientation' }
-      },
-      args
-    });
-  },
-  decorators: [
-    (story: () => typeof html) => html`
-      <style>
-        td.template {
-          width: 100%;
-          height: 250px;
-        }
-      </style>
-      ${story()}
-    `
-  ]
-};
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="space-y-5">
```

Comment:
This is more for general readability, but can we increase the gap between the two step groups here for the size story: 
<img width="966" alt="image" src="https://github.com/user-attachments/assets/ee77b5e6-09b8-4b06-b249-badef1447582">